assets/js/modules/analytics-4/datastore/settings.js

```diff
@@ -52,7 +52,7 @@ export async function submitChanges( { select, dispatch } ) {
 	}
 
 	const webDataStreamID = select( STORE_NAME ).getWebDataStreamID();
-	if ( ! isValidWebDataStreamID( webDataStreamID ) ) {
+	if ( webDataStreamID === WEBDATASTREAM_CREATE || ! isValidWebDataStreamID( webDataStreamID ) ) {
```

Comment:
Shouldn't this only happen if `webDataStreamID === WEBDATASTREAM_CREATE` like it does above for properties? It seems like this condition is accounting for this not being set somewhere.

Reply:
Updated the `submitChanges` function to use the `WEBDATASTREAM_CREATE` constant instead of an empty string, but left this condition as is.